Adding more details to BTC Dev workflow (#3081)
dfinity/portal: docs/developer-docs/multi-chain/overview.mdx

### Key management and signing

> A key feature of the ICP's Chain Fusion capabilities is [threshold signing](/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa/). This feature allows canisters to control accounts on other blockchains. The private keys that are used to sign transactions are never accesible in a single physical place, but securely distributed in form of shares among a group of nodes which perform a threshold signing protocol to generate a signature on behalf of the canister.

Reviewer: ```suggestion
A key feature of the ICP's Chain Fusion capabilities is [threshold signing](/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa/). This feature allows canisters to control accounts on other blockchains through private and public key pairs. The private keys are used to sign transactions, and are never accessible in a single physical place. Each key is securely distributed in shares among a group of nodes which perform a threshold signing protocol to generate each signature on behalf of the canister.
```
